lineman/app/components/thread_page/proposal_pie_chart/proposal_pie_chart.scss

```diff
@@ -50,8 +38,8 @@
   color: $grey-on-white;
 }
 
-@media (min-width: $medium-max-width) and (max-width: $large-max-width) {
-  .proposal-pie-chart__legend{
-    display: block;
+@media (max-width: $medium-max-width) {
+  .proposal-pie-chart__legend {
```

Comment:
FYI I'm planning to get in tomorrow and put a bit more design into how the
2 columns work on different screens. I think we should be able to have the
pie and legend sitting comfortably side by side on any screen.

On Tue, Oct 13, 2015 at 6:53 PM James Kiesel <notifications@github.com>
wrote:

> In
> lineman/app/components/thread_page/proposal_pie_chart/proposal_pie_chart.scss
> <https://github.com/loomio/loomio/pull/2578#discussion_r41829229>:
>
> > @@ -50,8 +38,8 @@
> >    color: $grey-on-white;
> >  }
> >
> > -@media (min-width: $medium-max-width) and (max-width: $large-max-width) {
> > -  .proposal-pie-chart__legend{
> > -    display: block;
> > +@media (max-width: $medium-max-width) {
> > +  .proposal-pie-chart__legend {
>
> This doesn't work for me between 760px and 840px width, and should be
> floating right from the pie chart from 480px - 760px. Not sure why we need
> to change this, actually.
>
> Might consider trying display: flex out here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2578/files#r41829229>.
>